src/hotspot/share/cds/filemap.cpp

```diff
@@ -2086,7 +2069,7 @@ address FileMapInfo::heap_region_requested_address() {
   } else {
     // We can avoid relocation if each region is mapped into the exact same address
     // where it was at dump time.
-    return /*dumptime*/header()->heap_begin() + r->mapping_offset();
+    return (address)ArchiveHeapWriter::NOCOOPS_REQUESTED_BASE;
```

Comment:
Can you please update the comment above this line as well to remove `each region` as we only have single heap region now.

Reply:
Thanks for the review. I updated the comment to clarify that this address was hard-coded at dump time, and explained the reason for that.